test/flop/schema_test.exs

```diff
@@ -167,7 +167,7 @@ defmodule Flop.SchemaTest do
 
     test "raises if sortable field is unknown" do
       assert_raise Flop.UnknownFieldError, fn ->
-        defmodule Marmelade do
+        defmodule Marmalade do
```

Comment:
That was the [German spelling](https://de.wikipedia.org/wiki/Marmelade). 😄

Reply:
That link is not German enough!

https://www.duden.de/rechtschreibung/Marmelade 👈🏻